lms/static/scripts/frontend_apps/index.js

```diff
@@ -72,8 +73,10 @@ if (!rootEl) {
   throw new Error('#app container for LMS frontend is missing');
 }
 render(
-  <Config.Provider value={config}>
-    <Services.Provider value={services}>{app}</Services.Provider>
-  </Config.Provider>,
+  <IntlProvider messages={{}} locale="en" defaultLocale="en">
```

Comment:
Ob'ly, we don't want to have to wrap all of our components in test like this, so we'll want to figure something out that's more convenient. But this does work for the mo.